TrackletAlgorithm/TrackletCalculator_calculate.h

```diff
@@ -0,0 +1,455 @@
+#ifndef TrackletAlgorithm_TrackletCalculator_calculate_LXLY_h
+#define TrackletAlgorithm_TrackletCalculator_calculate_LXLY_h
+#include "TrackletProjectionCalculator.h"
+#include "TrackletLUTs.h"
+
+
+// This function does all the actual calculations in the TrackletCalculators
+// for the barrel-only seeds (L1L2, L2L3, L3L4, and L5L6).
```

Comment:
@aryd I think we discussed adding the PDF documenting these calculations to the Wiki of this repo. If I remember correctly, could you please do that and add a link to it here?

Reply:
The git wiki does not seem to allow attachments. Do you have a suggestion for where I should put the actual pdf file ?

Reply:
Ok - figured out how to do this.